azerothcore/azerothcore-wotlk
# chore(Core/Player): remove useless SendQuestComplete

imported from Maczuga/SunwellCore@5d9826c

## CHANGES PROPOSED:
-  remove useless SendQuestComplete

## ISSUES ADDRESSED:
- related #1141

## TESTS PERFORMED:
- didn't test

## HOW TO TEST THE CHANGES:
- check quest completion of several quests, do you see the question mark and that the quest is completed? can you complete and get the reward?

## Target branch(es):








 
## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
